noaa.go

```diff
@@ -323,3 +382,23 @@ func GridpointForecast(lat string, long string) (forecast *GridpointForecastResp
 	forecast.Point = point
 	return forecast, nil
 }
+
+// HourlyForecast returns an array of raw hourly forecast data
+func HourlyForecast(lat string, long string) (forecast *HourlyForecastResponse, err error) {
```

Comment:
An hourly forecast is an awesome addition. Thank you for putting this in the module.

Reply:
I needed this for work I am doing.  I suppose you could say that I'm scratching my itch.

Your code already addressed much of what I needed... it made no sense to me to rewrite it when I could more easily add this feature.  Pulling JSON objects into their own structs wasn't strictly necessary, but it helps simplify using the library.  I'll adjust the code per your request and commit.